docs/mkdocs/docs/developers/paig-client/code-examples/milvus-integration/a-sample-tutorial.md

```diff
@@ -0,0 +1,253 @@
+---
+title: Sample Tutorial
+---
+
+# Milvus Integration Sample Tutorial
+
+This tutorial shows you how to prevent unauthorized access to your data by:
+
+- Restricting access based on users and groups.
+- Restricting access based on metadata policies.
+
+## Prerequisite tasks
+
+### Install Docker 
+Make sure you have Docker installed on your system. You can download it from the [Docker website](https://www.docker.com/products/docker-desktop/).
+
+### Set up the tutorial directory
+Download below files and place them in new directory where you want to run the tutorial.
+
+ - <a href="files/docker-compose.yml" download="docker-compose.yml">docker-compose.yml</a>
+ - <a href="files/requirements.txt" download="requirements.txt">requirements.txt</a>
+ - <a href="files/create_milvus_collection.py" download="create_milvus_collection.py">create_milvus_collection.py</a>
+ - <a href="files/query_milvus_collection.py" download="query_milvus_collection.py">query_milvus_collection.py</a>
+
+### Start Milvus and Attu server
+!!! note "Note"
+    If you have your own Milvus server running and you want to use that, you can skip this step.
+
+Run the following command in the directory where you placed the files.
+    ```bash
+    docker-compose up -d
```

Comment:
we can have two paths, using existing Milvus server or starting new one.

Under new one, docker install, docker-compose file download and command can be put

Reply:
Added all the docker related things in single step with note so they can skip if they want to use their own